### DISCUSSION TITLE
Verfügbarkeit von neuen Firmwareversionen

### DISCUSSION THREAD

**Discussion by kaimimue** [SukramJ/aiohomematic#1724] [Q&A] [answered]
Es ist möglich über HomeAssistant die Geräte upzudaten, Dazu muss ich die Firmware vorher auf die CCU/RaspberryMatic hochladen. Dies funktioniert auch soweit. Allerdings habe ich keine Möglichkeit gefunden über dir Verfügbarkeit von neuen Versionen in Homeassistant informiert zu werden:

![image](https://github.com/user-attachments/assets/45121338-93d2-4d95-b303-ef9d7165528d)

Ist es möglich diese Benachrichtigung irgendwie in HomeAssistant abzufragen?

**Comment by SukramJ** [comment, marked as answer]
> Ist es möglich ...

Nein.

Na, wo hast Du die Frage sonst noch gestellt?
